orphedomos-maven-plugin/src/main/java/org/technologybrewery/orphedomos/mojo/RetrieveCachedWheelsMojo.java

```diff
@@ -0,0 +1,22 @@
+package org.technologybrewery.orphedomos.mojo;
+
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.technologybrewery.habushu.RetrieveWheelsMojo;
+
+/**
+ * Helper mojo that handles the retrieving of wheel artifacts from poetry
+ * cache by artifactId and into a given targetDirectory during the 
+ * {@link LifecyclePhase#VALIDATE} build phase. 
+ *
+ * @param wheelDependencies A List of Wheel Dependencies which will identify wheel 
+ *                          files by {@WheelDependency.artifactId} in poetry cache and place them into 
+ *                          a given {@WheelDependency.targetDirectory}. This logic specifically targets
+ *                          wheel artifacts cached by the {@param cacheWheels} parameter and REQUIRES 
+ *                          the requested wheel to have first been cached prior to setting this config
+ * @throws HabushuException
+ */
+@Mojo(name = "retrieve-wheels", defaultPhase = LifecyclePhase.VALIDATE)
+public class RetrieveCachedWheelsMojo extends RetrieveWheelsMojo {
+    
```

Comment:
It looks like you are just extending a new in Orphedomos from Habushu but since its identical, why not just use the mojo from Habushu since your not adding any new functionality?

Reply:
I imagine this is done to get it to run in the Validate lifecycle

Reply:
Originally Habushu's logic insisted on the calling project's packaging to be "habushu" only in order to execute otherwise it would not run. However, the 2nd code change was needed to bypass this logic so actually I think I could probably achieve my goal without this orphedomos change. Let me consider.

Reply:
Yes it looks like it'll be best for this functionality to be extended in orphedomos so i'll live it here. @cpointe-ibllanos @jacksondelametter